app/views/notifications/edition_published.text.erb

```diff
@@ -1,6 +1,8 @@
-Hi,
+Hello,
 
-The <%= @edition.format_name %> '<%= @edition.title %>' has now been published.  It is now available at the following url: <%= @public_url %>.
+The <%= @edition.format_name %> '<%= @edition.title %>' is published. It's now available at the following url: <%= @public_url %>
+
+New content will be live immediately, changes to previously published content can <a href="https://insidegovuk.blog.gov.uk/2014/11/05/how-soon-do-users-see-content-after-youve-pressed-publish/">take up to 15 minutes</a>. 
```

Comment:
As this is a template for a plain text email, making a link like that won't work. It's probably best to put the URL in the text at the end, as with the line above.